client/src/document/ingredients/browser-compatibility-table/legend.tsx

```diff
@@ -73,7 +74,14 @@ function getActiveLegendItems(
         if (versionSupport.prefix) {
           legendItems.add("prefix");
         }
-        if (versionSupport.notes) {
+        if (versionSupport.notes && Array.isArray(browserSupport)) {
```

Comment:
What does it mean for a feature if `Array.isArray(browserSupport)`?

Reply:
It means there is more than one item for it. Basically, it means its functionality has changed at some point, so there is information about older browsers included. I don't like the code I added to the legend myself if I'm honest; it's an adaptation that attempts to match when the symbols appear, but it's sort of a brute force way of doing so because I don't want to run the same checks for the legend that are run for each row. In a separate branch, I was trying to overhaul the entire legend so that it just checks if each icon appears in the table (basically `if (document.querySelector(".bc-table .icon-yes")` etc.), but I ran into the issue that this code runs before the table fully exists.

Reply:
Remember, this is inside a loop that loops through elements of `asList(browserSupport)`. This basically means more than one such element exists (there are at least two list items for this browser), and there are notes. It's not exactly the same check as the feature-row: the exact same check would be a check outside the loop that says `!getFirst(browserSupport)?.notes?.length && asList(browserSupport).some((s) => s && s.hasOwnProperty("notes"))`